pkg/renderer/readme.go

```diff
@@ -48,6 +48,7 @@ the login page, and log in with username "admin" and the password found in the
      If you want to run the StackRox Scanner:
```

Comment:
Again, while we are at it, maybe you can also improve the wording here a little bit.
As I understand it "StackRox Scanner" in the docs refers to what we are currently calling "Scanner V2" while "Scanner V4" is the name of the new scanner.

Hence, I would suggest something along the lines:
```
- Deploy Scanner Components
  - Run scanner/scripts/setup.sh
  - Run {{.K8sConfig.Command}} create -R -f scanner
  - Run scanner-v4/scripts/setup.sh
  - Run {{.K8sConfig.Command}} create -R -f scanner-v4
```

Note that I have deliberately removed the line "If you want to ...", because it is my impression that a Central installation without scanner(s) is quite outside the norm. Not sure to what degree we even support such scenarios. @dcaravel Could you comment on this? Is it ok to let the README within the deployment bundle convey the message that "installing scanner" isn't really just an optional step?

Reply:
I was also wondering what happens if Scanner is not installed at all.

Should we however mention that scanner-v4 is optional? You can install `scanner` and skip `scanner-v4`, but  the other way around isn't supported.

Reply:
No, at the moment we should not mention Scanner V4 being optional. That is the situation as of now (pulling in Scanner V4 by default + unofficial escape hatch).

Reply:
> @dcaravel Could you comment on this? Is it ok to let the README within the deployment bundle convey the message that "installing scanner" isn't really just an optional step?

Yes I would consider a no scanner installation outside the norm, however may be some bias there based on the helm/operator defaults and that most engagements that come our way are scanner related.  But 'technically' its still an option to not install it.

> No, at the moment we should not mention Scanner V4 being optional. That is the situation as of now (pulling in Scanner V4 by default + unofficial escape hatch).

FWIW, was decision this week to include the 'escape hatch' instructions in the official docs (refer to comments in ROX-19804).